packages/button/src/Button.tsx

```diff
@@ -175,24 +175,24 @@ const buttonVariants: { readonly [K in Variant]: string } = {
 
 const buttonSizes: Record<Size, string> = {
   [Size.XSmall]: css`
-    height: 22px;
+    min-height: 22px;
```

Comment:
Made this min-height so that when the text was overflowing, the button grew. I did this to account for removing the `overflow: hidden` rule. If we don't like this approach it's easy to add back the `overflow:hidden` rule; however, apply it to the span contained inside of the button rather than to the button itself.

Reply:
👍